packages/x-data-grid-pro/src/tests/columns.DataGridPro.test.tsx

```diff
@@ -473,7 +473,7 @@ describe('<DataGridPro /> - Columns', () => {
       render(<Test rows={rows} columns={columns} />);
       await apiRef.current.autosizeColumns();
       await microtasks();
-      expect(getWidths()).to.deep.equal([213, 235]);
+      expect(getWidths()).to.deep.equal([155, 177]);
```

Comment:
I don't get why it fails in Circle CI, it passes locally on my machine.

<img width="942" src="https://github.com/mui/mui-x/assets/13808724/7a6c87a8-acaf-4713-a91a-e0ba3e08f366">

> https://app.circleci.com/pipelines/github/mui/mui-x/54728/workflows/1766d828-40a1-44dc-a047-9c2720fa5c26/jobs/313068

I have no idea where these extra 58px are coming from:

<img width="316" src="https://github.com/mui/mui-x/assets/13808724/7c39a517-6005-4711-ba39-3406487e0b31">

I can keep the previous values to make CI pass, but:
- these tests would fail for me locally
- these previous values feel wrong to me (see the screenshot above)

@romgrk any ideas?

Reply:
Weird, but I have no idea, don't remember this specific case.

Reply:
I kept the old values to make it pass in CI 🙃